pkg/controller/manifestwork/manager.go

```diff
@@ -99,7 +100,12 @@ func add(importSecretInformer cache.SharedIndexInformer, mgr manager.Manager, r
 	if err := c.Watch(
 		&runtimesource.Kind{Type: &clusterv1.ManagedCluster{}},
 		&handler.EnqueueRequestForObject{},
-	); err != nil {
+		predicate.Predicate(predicate.Funcs{
+			GenericFunc: func(e event.GenericEvent) bool { return isDefaultModeObject(e.Object) },
```

Comment:
do we need this? I saw we seem already added the judgement in the controller

Reply:
The judgment in the controller is in order to process requests from import secrets, functionally we can remove the predicate here, but it can filter the managed cluster earlier, so I added.